docs/content/integrations/spark.mdx

```diff
@@ -1,62 +1,60 @@
+The advantage of this approach is a very clean local testing story. You can run an entire pipeline of Spark assets in a single process. You can use [IO managers](/concepts/io-management/io-managers) to abstract away IO - storing outputs on the local filesystem during local development and in the cloud in production.
 
 The downside is that this approach only works with PySpark, and setting up a step launcher can be difficult. We currently provide an <PyObject module='dagster_aws.emr' object='emr_pyspark_step_launcher' /> and a <PyObject module='dagster_databricks' object='databricks_pyspark_step_launcher' />, but if you need to submit your Spark job to a different kind of cluster, writing your own can be time consuming ([here are some tips](https://github.com/dagster-io/dagster/discussions/3201)). You also need to install the Dagster library itself on the cluster.
 
-### Running PySpark code in ops
+### Running PySpark code in assets
 
 <CodeReferenceLink filePath="examples/with_pyspark" />
 
-Passing PySpark DataFrames between ops requires a little bit of extra care, compared to other data types, for a couple reasons:
+Passing PySpark DataFrames between asset functions requires a little bit of extra care, compared to other data types, for a couple reasons:
 
 - Spark has a lazy execution model, which means that PySpark won't process any data until an action like `write` or `collect` is called on a DataFrame.
-- PySpark DataFrames cannot be pickled, which means that [IO Managers](/concepts/io-management/io-managers) like the `fs_io_manager` won't work for them.
+- PySpark DataFrames cannot be pickled, which means that [IO Managers](/concepts/io-management/io-managers) like the default filesystem IO manager won't work for them.
 
 In this example, we've defined an <PyObject module='dagster' object='IOManager' /> that knows how to store and retrieve PySpark DataFrames that are produced and consumed by ops.
 
-This example assumes that all the outputs within the dagster job will be PySpark DataFrames and stored in the same way. To learn how to use different IO managers for different outputs within the same dagster job, take a look at the [IO Manager](/concepts/io-management/io-managers) concept page.
+This example assumes that all the outputs within the Dagster pipeline will be PySpark DataFrames and stored in the same way. To learn how to use different IO managers for different outputs within the same Dagster pipeline, take a look at the [IO Manager](/concepts/io-management/io-managers) concept page.
 
-This example writes out DataFrames to the local file system, but can be tweaked to write to cloud object stores like S3 by changing to the `write` and `read` invocations.
+This example writes out `DataFrames` to the local file system, but can be tweaked to write to cloud object stores like S3 by changing to the `write` and `read` invocations.
```

Comment:
```suggestion
This example writes out `DataFrames` to the local file system, but can be tweaked to write to cloud object stores like Amazon S3 by changing to the `write` and `read` invocations.
```

The first time a branded product is mentioned, the name of the company/creator/provider/etc should be include. Like `Google BigQuery`, thereafter just `BigQuery`.